### PR TITLE
Add 2021q3 Quarterly Status Reports for Intel and RTW88 wireless.

### PR DESCRIPTION
**Pull request by bzfbd**

### CODE REVIEW

File: 2021q3/rtw88.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Links: +
+link:https://wiki.freebsd.org/WiFi/Rtw88[Rtw88 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw88[https://wiki.freebsd.org/WiFi/Rtw88]
+
+Contact: Bjoern A. Zeeb <bz@FreeBSD.ORG>
+
+This project aims to bring in support for Realtek's rtw88 chips.
+
+With the growing LinuxKPI compatibility code an initial port of the Realtek rtw88 driver was done.
+This gives us a second driver symptome after Intel and helps to validate and grow the LinuxKPI code base.
```
Comment (pauamma) [inline 2021q3/rtw88.adoc:11]: "symptome"? Do you mean "support for a second driver"?

File: 2021q3/rtw88.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Changes and overall time needed to get the driver compiling were very little.
+At the moment we are seeing DMA issues which prevent most people from loading firmware or later on using the driver.
```
Comment (pauamma) [inline 2021q3/rtw88.adoc:13]: I'd move "later on" to the end of the sentence.

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2021q3/rtw88.adoc (hunk 1 above)
Comment (bzfbd) [reply 2021q3/rtw88.adoc:11]: Lookup "second system syndrome"  I'll change it.

**Comment by bzfbd** [comment]
Okay I added another commit;  is there a way to fold them together without forcefully pushing an update?

File: 2021q3/rtw88.adoc (hunk 1 above)
Comment (pauamma) [reply 2021q3/rtw88.adoc]: Oh, OK.Thanks. (No need for me to look it up, I have it in cache from the last time I read TMMM.)

**Review by pauamma — APPROVED** [review]
LGTM.

**Comment by pauamma** [comment]
> Okay I added another commit; is there a way to fold them together without forcefully pushing an update?

There's a dropdown field called "Changes from" (just below the Conversation/Commits/Checks/Files changed/ - see https://github.com/freebsd/freebsd-quarterly/pull/407/files/7066f058589bc432a1aeeaa79f5552dbfddf0bd5..e47d316fa5aeafcafaf79a28b60762545383f0eb) that you can set to "All changes" to do what I think you're asking.

**Pull request merged**